integration-tests/tests/package.json

```diff
@@ -2,17 +2,18 @@
 	"name": "@realm/integration-tests",
   "version": "0.1.0",
   "description": "A set of tests that can run in different environments",
+  "type": "module",
   "main": "src/index.ts",
   "exports": {
     ".": "./src/index.ts",
     "./node": "./src/node/index.ts"
   },
   "private": true,
   "scripts": {
-    "start": "CONTEXT=appTemplatesPath=../realm-apps,$CONTEXT node start.js --watch",
+    "start": "CONTEXT=appTemplatesPath=../realm-apps,$CONTEXT node start.mjs --watch",
     "start:missing-server": "CONTEXT=missingServer,$CONTEXT npm start",
     "start:performance": "CONTEXT=missingServer,performance,integration=false,$CONTEXT npm start",
-    "test": "CONTEXT=appTemplatesPath=../realm-apps,$CONTEXT node start.js --exit",
+    "test": "CONTEXT=appTemplatesPath=../realm-apps,$CONTEXT node start.mjs --exit",
```

Comment:
I had to switch the `tests` package to a module to make it load the right `@realm/network-transport` bundle. (at least I think that was why).